src/lib/native.ts

```diff
@@ -343,8 +343,21 @@ export async function temp(content: string, prefix: string) {
     })
 }
 
-export async function move(from: string, to: string) {
-    return sendNativeMsg("move", { from, to }).catch(e => {
+export async function move(
+    from: string,
+    to: string,
+    overwrite: boolean,
+    cleanup: boolean,
+) {
+    const requireNativeMessengerVersion = "0.3.0"
+
+    if (!(await nativegate(requireNativeMessengerVersion, false))) {
+        throw new Error(
+            `'restart' on Windows needs native messenger version >= ${requireNativeMessengerVersion}.`,
```

Comment:
: )

Reply:
You are welcome. 🙃

Reply:
@bovine3dom Fixed in https://github.com/gsbabil/tridactyl/commit/c17ac418266995de2f2540ee1178ee79eb41074a